connector/src/main/java/org/geysermc/connector/sound/SoundMap.java

```diff
@@ -0,0 +1,112 @@
+package org.geysermc.connector.sound;
```

Comment:
Missing license header, and please relocate this to the `translators` packet. Overall, this class should be formatted in a similar way to that of other translators as currently it's inconsistent with the current codebase.